cli/build.rs

```diff
@@ -122,10 +122,16 @@ mod ts {
 
   deno_core::extension!(deno_tsc,
     ops = [op_build_info, op_is_node_file, op_load, op_script_version],
+    esm_entry_point = "ext:deno_tsc/99_main_compiler.js",
+    esm = [
+      dir "tsc",
+      "97_ts_host.js",
+      "98_lsp.js",
+      "99_main_compiler.js",
```

Comment:
Won't this make it load slower? I think we should bundle this.

Reply:
This will be negligible since it's already snapshotted, in general I don't think this will be visible in any way, but I can provide some number if you'd like.

Reply:
It's ok. Based on divy's snapshotting tool, it adds 0.04ms per file. Still think we should bundle this though, but that's for a different day.